main.nf

```diff
@@ -11,18 +11,39 @@
 
 nextflow.enable.dsl = 2
 
+// We can't change global parameters inside this scope, so we build the ones we need locally
+def edyeet_merge_cmd=params.edyeet_merge_cmd
```

Comment:
Maybe what works is the following:

Rewrite the parameter definitions below into:
`edyeet_merge_cmd = "-M" ? params.edyeet_merge_cmd : ''`

You can apply such a pattern to all of them.

Reply:
Replace the stuff behind the colon with your default of choice.

Reply:
Thanks for the ternary operator tip. The actual code looks a little bit different, though. Please see for yourself.

Reply:
![image](https://user-images.githubusercontent.com/21954664/100904090-f3008a80-34c6-11eb-84cd-ad3f728b23d6.png)